### PR TITLE
Added HavlockV/CoC support

### PR DESCRIPTION
**Pull request by lozalojo**
Added HavlockV/CoC support, BUT:

The system has a bug so that does not update the mov.value automatically reported here:

https://github.com/HavlockV/CoC7-FoundryVTT/issues/593

Until its fixed, drag-ruler will work only on manually entered mov values.

But the mov attribute I entered at the system.js is correct

### CODE REVIEW

File: src/systems.js [hunk 1]
```diff
@@ -15,7 +15,9 @@ export function getDefaultSpeedAttribute() {
 		case "shadowrun5e":
 			return "actor.data.data.movement.walk.value";
 		case "swade":
-			return "actor.data.data.stats.speed.value"
+			return "actor.data.data.stats.speed.value";
+		case "CoC7":
```
Comment (manuelVo) [inline src/systems.js:19]: Please move this up so it follows an alphabetical order.

File: src/systems.js [hunk 2]
```diff
@@ -31,7 +33,9 @@ export function getDefaultDashMultiplier() {
 		case "D35E":
 		case "sfrpg":
 		case "shadowrun5e":
-			return 2
+			return 2;
+		case "CoC7":
```
Comment (manuelVo) [inline src/systems.js:37]: I guess this one sits fine. I kinda like the order by returned number here :)

**Review by manuelVo — CHANGES_REQUESTED** [review]
Thanks. Please adjust the ordering of the values and then this will be ready for merge.

**Comment by lozalojo** [comment]
Reordered to match alphabetical order, thou the current version wasnt... added semicolon to some options.

There is a PR to fix the bug I mentioned at the system's repo, so the fix its on its way:
https://github.com/HavlockV/CoC7-FoundryVTT/pull/591

**Comment by manuelVo** [comment]
Merged as 530a16ca5d0045fc32dd5fdf2df3979ab538dda7

Somehow during your last commit the file got completely mixed up, assigning the wrong value for the pf1 system. I've cleaned that up and removed any changes that didn't belong to this original PRs intent. Thanks for adding those values in :)

**Comment by lozalojo** [comment]
Ok, thank you, I see pf1 does not assign any value neither in the old or the new version, and to match alphabetical order D35E should go just after CoC, but it is ok by my side.

**Comment by manuelVo** [comment]
What pf1 does is what's called a case fallthrough: There is no break statement after pf1. As a result it executes the code of the case-statement below. So pf1 and D35E get the same value assigned.

**Comment by lozalojo** [comment]
> What pf1 does is what's called a case fallthrough: There is no break statement after pf1. As a result it executes the code of the case-statement below. So pf1 and D35E get the same value assigned.

I have to learn js, thank you!

**Pull request closed without merging**